src/useDragSelection.js

```diff
@@ -95,128 +84,143 @@ function useDragSelection(targetRef, onSelectionChange) {
     setSelectionBox(selectionBox);
   }, [dragStart, dragEnd]);
 
-  const getTargetMargin = (target) => {
+  function getTargetMargin(target) {
```

Comment:
Any specific reason to convert this in plain `function`? If not, can we keep the function declaration consistent?

Reply:
Sure, we just need to move the declaration to the top of the component in that case.